Clarify that osu!mania hitsounding exemptions are for hitsound additions (#7503)
ppy/osu-wiki: wiki/Beatmapping/Hitsound/en.md

**Hitsounds** are the sounds that osu! plays in response to user input when interacting with [hit objects](/wiki/Hit_object). Hitsounds are typically used to give auditory feedback to the player to help them judge their [accuracy](/wiki/Gameplay/Accuracy) in relation to the song.

> Hitsound feedback consists of a default sample called a *hitnormal*, and any combination of whistle, finish, or clap sample *additions*. Each sample has one of three "sample sets" that change its style: `Normal`, `Soft`, or `Drum`.

Reviewer: ```suggestion
Hitsound feedback consists of a default sample called a *hitnormal*, and any combination of whistle, finish, or clap sample *additions*. Each sample is part of one of three *sample sets* that denote different styles: `Normal`, `Soft`, or `Drum`.
```

Author: not even related to pr but :ok:  :100:
